### PR TITLE
Add empty webui + rework port config

### PR DESCRIPTION
**Pull request by bajtos**
- Rework the HTTP server to serve a dummy HTML page at `/webui` path
- Rework port configuration to use port number 5500 by default and allow
  the caller to change the port number via `PORT` environment variable.

### CODE REVIEW

**Comment by bajtos** [comment]
@aarshkshah1992 please review. I slightly tweaked your initial web server to get it closer to what I envision for the Station Electron app.

File: main/main.go [hunk 1]
```diff
@@ -35,27 +42,16 @@ func main() {
 	}()
 
 	time.Sleep(1 * time.Second)
```
Comment (bajtos) [inline main/main.go:44]: @aarshkshah1992 Why is this sleep needed? Are we trying to wait for the HTTP server to start? I believe the listener `nl` is already accepting new TCP connections by now.

Comment (aarshkshah1992) [reply main/main.go:44]: @bajtos 

You are right, we can remove this.

**Review by aarshkshah1992 — APPROVED** [review]
lgtm

File: main/main.go (hunk 1 above)
Comment (bajtos) [reply main/main.go:44]: Cool, I opened a follow-up PR:
https://github.com/filecoin-project/saturn-l2/pull/16

@aarshkshah1992 PTAL

**Pull request merged**